k6/src/lib/common.js

```diff
@@ -264,7 +282,16 @@ function TestScenarioBuilder() {
       run: function (testParameters, iteration = 0) {
         const response = that._request(testParameters, iteration);
         check(response, that._checks);
-      },
+
+        // if Load test, then we need to sleep for random time between 1 and 5 seconds
```

Comment:
Q: why?
Also can remove the random extra new lines

Reply:
✅  Done